src/app/concrete_dst_router.rs

```diff
@@ -0,0 +1,206 @@
+use futures::Poll;
+use http;
+use std::fmt;
+use std::marker::PhantomData;
+use std::time::Duration;
+
+use never::Never;
+
+use svc;
+use super::dst::DstAddr;
+use proxy::http::settings;
+use proxy::http::profiles::ConcreteDst;
+
+extern crate linkerd2_router as rt;
+
+pub use self::rt::{error, Router};
+
+#[allow(dead_code)]
```

Comment:
I'm guessing this `allow` is necessary as the compiler is not able to determine this is used? In that case, it would be nice to have a comment stating that.